lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -41,7 +33,19 @@
         padding: $base-spacing;
       }
     }
+  }
+
+  .card__copy {
+    &.-share {
+      h3:last-of-type,
```

Comment:
Hmm, I'm a little wary about using `:last-of-type` here in case we were to add more markup in sometime later. I'm wondering if actually adding some global utility classes like `.show-only-mobile`, `.show-only-tablet` might be a better approach? (Sorry to be so picky/indecisive.)